apps/desktop/src/app/upload/upload.component.spec.ts

```diff
@@ -38,8 +45,11 @@ describe('UploadComponent', () => {
         ReactiveFormsModule,
         MatDialogModule,
         ConfirmFilesModule,
-        PaButtonModule,
-        PaTextFieldModule,
+        MockModule(PaButtonModule),
```

Comment:
Just to know, is `MockModule` required for pastanaga modules ?

Reply:
no it's not required. It's just a way to make test more performant by not loading everything for real. That being said, I didn't check the performances before/after using MockModule here, I just added them by habit.

Reply:
Thanks for the info